lib/assessment.js

```diff
@@ -8,40 +8,64 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue) {
+  return previousValue < currentValue? previousValue : currentValue;
 };
 
+
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
-
+var sum = function sum(previousValue, currentValue) {
+  return previousValue += currentValue;
```

Comment:
This will work correctly, but assigning to a parameter is frowned upon except when setting a default value or in rare cases where it's the way to go. Returning `previousValue + currentValue` was an ideal solution; this works because the two lines evaluate to the same thing, and so the return value is the same.

1